content/blog/end-to-end-easy-to-use-pipeline-for-training-a-model-on-medmnist-v2-using-hpe-machine-learning-development-environment-flask.md

```diff
@@ -0,0 +1,514 @@
+* Upon completion of the trial. 
+
+Additional checkpoint configuration settings can be modified to make this even more customizable. Checkpoints can be easily examined through the WebUI even after an experiment completes, and downloaded easily through the Determined APIs: 
+
+![](/img/screenshot3.png)
+
+#### Hyperparameter search 
+
+Hyperparameter search refers to the process of searching for the optimal configuration settings for your machine learning model – for example, searching for the optimal learning rate or convolution sizes in a convolutional neural network. The original training script does not implement hyperparameter search. This is not surprising, as hyperparameter search is yet another heavy lift for a researcher who wants to focus on experimentation, not the grunt work of setting up hyperparameter search.  
+
+With Determined, configuring hyperparameter search is easy. Defining hyperparameters, either static or in ranges, is easy through experiment configuration: 
+
+```yaml
+hyperparameters:
+    global_batch_size: 128
+    data_flag: pathmnist
+    dataset_name: "pathmnist.npz"
+    model_flag: resnet18
+    lr: 0.001
+    gamma: 0.1
+    resize: True
+    task: "multi-class"
+    num_epochs: 15
+```
+
+In this case, the “hyperparameters” feature can also be used to switch between datasets and types of training tasks if configured appropriately in the training script – which is neat! 
+
+#### Collaboration 
+
+Data scientists and researchers rarely work alone, especially in today’s data-driven technological boom. Sharing experiment results across a large team and sharing a single GPU cluster is not traditionally straightforward, but HPE Machine Learning Development Environment makes for a much better experience: 
+
+#### Resource Management 
+
+At the enterprise level, HPE Machine Learning Development Environment automatically scales up and down workloads depending on priority and resource availability. For example, if a researcher is running a hefty training job and is utilizing all 10 GPUs on a shared cluster, but their colleague needs two of them for a higher priority smaller job, HPE Machine Learning Development Environment can temporarily scale back the larger job utilizing all 10 GPUs down to using only 8, leaving room for the smaller training job.  
+
+#### Model Sharing and Reproducibility 
+
+The HPE Machine Learning Development Environment WebUI makes it easy to track experiments and see which model configurations resulted in particular results, across a team. This makes reproducibility easy, something that is of utmost importance when developing models for a use case like predicting cancer in biomedical images.  
+
+### Appendix: Building an E2E solution 
+
+#### Step 1: Model Training 
+
+To get started, we’ll take a look at the original training script provided [here](https://github.com/MedMNIST/experiments/blob/main/MedMNIST2D/train_and_eval_pytorch.py). 
```

Comment:
As mentioned earlier, avoid using "we'll" and instead phrase it as:

"To get started, take a look at the original training script provided [here]."